### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jprodriguezg**
Dear admin,

Please consider these contributions for the repository.

Best regards,
Juan Pablo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,6 +301,9 @@ arXiv, 2022. [YouTube](https://youtu.be/HV9_FhS-f88), [code](https://uzh-rpg.git
 - <a name="Ercan23cvprw"></a>Ercan, B., Eker, O., Erdem, A., Erdem, E.,  
 *[EVREAL: Towards a Comprehensive Benchmark and Analysis Suite for Event-based Video Reconstruction](https://openaccess.thecvf.com/content/CVPR2023W/EventVision/papers/Ercan_EVREAL_Towards_a_Comprehensive_Benchmark_and_Analysis_Suite_for_Event-Based_CVPRW_2023_paper.pdf)*,  
 IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2023. [PDF](https://openaccess.thecvf.com/content/CVPR2023W/EventVision/papers/Ercan_EVREAL_Towards_a_Comprehensive_Benchmark_and_Analysis_Suite_for_Event-Based_CVPRW_2023_paper.pdf), [Project Page](https://ercanburak.github.io/evreal.html), [Suppl.](https://openaccess.thecvf.com/content/CVPR2023W/EventVision/supplemental/Ercan_EVREAL_Towards_a_CVPRW_2023_supplemental.zip), [Code](https://github.com/ercanburak/EVREAL).
+- <a name="Tapia23iros"></a>Tapia, R., Rodríguez-Gómez, J.P., Sanchez-Diaz, J.A., Gañán, F.J., Rodríguez, I.G., Luna-Santamaria, J., Martínez-De Dios, J.R., Ollero, A.,  
+*[A Comparison Between Framed-Based and Event-Based Cameras for Flapping-Wing Robot Perception](https://ieeexplore.ieee.org/document/10342500)*,
```
Comment (guillermogb) [inline README.md:305]: DOI link is preferred https://doi.org/10.1109/IROS55552.2023.10342500

File: README.md [hunk 2]
```diff
@@ -1126,6 +1129,9 @@ IEEE Int. Symp. Circuits and Systems (ISCAS), 2014. [YouTube](https://youtu.be/T
 - <a name="Rodriguez22icraw"></a>Rodriguez-Gomez, J.P., Gallego, G., Martinez-de Dios, J.R., Ollero, A.,  
 *[Stabilizing Event Data on Flapping-wing Robots for Simpler Perception](https://zenodo.org/record/6602940/files/Workshop_Flapping_Wing_Challenges.pdf)*,  
 IEEE Int. Conf. Robotics and Automation (ICRA) Workshop on Challenges of Flapping-wing Aerial Robots, 2022. [Slides](https://grvc.us.es/newweb/wp-content/uploads/2022/06/icra2022_flappingwingworkshop.pdf)
+- <a name="Rodriguez24ral"></a>Rodrıguez-Gomez, J.P., Martınez-de Dios, J.R., Ollero, A., Gallego, G.,  
+*[On the Benefits of Visual Stabilization for Frame- and Event-based Perception](https://ieeexplore.ieee.org/document/10648753/)*,
```
Comment (guillermogb) [inline README.md:1133]: DOI link: https://doi.org/10.1109/LRA.2024.3450290

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jprodriguezg) [reply README.md]: Ok thanks @guillermogb

**Review by guillermogb — APPROVED** [review]
Thank you, Dr. Rodrıguez-Gomez

**Pull request merged**